tests/test_xdsl_op_correctness.py

```diff
@@ -64,6 +64,4 @@ def test_u_and_v_conversion():
     norm_v2 = norm(v)
 
     assert np.isclose(norm_u, norm_u2, atol=1e-5, rtol=0)
-    assert np.isclose(norm_u, 26.565891, atol=1e-5, rtol=0)
```

Comment:
What is the norm now? Can you add this back with the right number?

Reply:
Just felt strange to me, added it back, it's now 2.0664787

Reply:
2.0664592 after switching back to time_order 2 :wink: